modules/nw-metallb-example-addresspool.adoc

```diff
@@ -19,9 +19,10 @@ metadata:
   namespace: metallb-system
 spec:
   addresses:
-  - 192.168.100.0/24
```

Comment:
Should I create a new example for the multiple specific IP address and keep the complete network range example?